pkg/crc/services/dns/dns.go

```diff
@@ -60,8 +69,8 @@ func RunPostStart(serviceConfig services.ServicePostStartConfig) (services.Servi
 
 	orgResolvValues := network.GetResolvValuesFromInstance(serviceConfig.Driver)
 	// override resolv.conf file
-	searchdomain := network.SearchDomain{Domain: serviceConfig.BundleMetadata.ClusterInfo.BaseDomain}
-	nameserver := network.NameServer{IPAddress: serviceConfig.HostIP}
+	searchdomain := network.SearchDomain{Domain: fmt.Sprintf("%s.%s", serviceConfig.Name, serviceConfig.BundleMetadata.ClusterInfo.BaseDomain)}
+	nameserver := network.NameServer{IPAddress: "10.88.0.8"}
```

Comment:
?

Reply:
This is the container IP which we can provide in runtime. can be constant, didn't do much refactor here.